Apps/docker-compose.yml

```diff
@@ -1,50 +1,51 @@
 version: '3'
 
 services:
-  backend:
-    depends_on:
-      - frontend
-    image: sosci/backend
+  generator:
+    image: sosci/generator
     build: 
-      context: ./backend
-      dockerfile: ./Dockerfile
-    
+      context: ./generator
+      dockerfile: Dockerfile
     restart: always
-
     environment:
       - HOST=0.0.0.0:34254
-      - TARGET=frontend:34255
+      - TARGET=backend:34255
       - FREQUENCY=1.0
+    ports:
+      - 34254:34254
     networks:
       - internal
 
-    expose:
-      - 34254
-
   frontend:
     image: sosci/frontend
+    depends_on:
+      - backend
     build: 
       context: ./frontend
-      dockerfile: ./Dockerfile
-    environment:
-      - HOST_PORT=34255
-      - HOST_ADDRESS=0.0.0.0
-
-    container_name: frontend
+      dockerfile: Dockerfile
     restart: always
-
-
+    ports:
+      - 5000:80
     networks:
       - internal
       - web
 
-    ports:
-      - 5000:80
+  backend:
```

Comment:
Would it be smart to introduce `depends_on: generator` for backend? (Genuinely don't know if that'd clever)